liquibase/changeLogs/capture/tables/datumConvertedValues/datumConvertedValues.sql

```diff
@@ -0,0 +1,12 @@
+create table ${AQTS_SCHEMA_NAME}.datum_converted_values
```

Comment:
As a double insurance policy - I know you already have the precondition set in the changeset that creates this table, but let's `if not exists` to this create table statement as well.